pkg/syncer/cnsoperator/controller/cnsregistervolume/cnsregistervolume_controller.go

```diff
@@ -292,6 +307,47 @@ func (r *ReconcileCnsRegisterVolume) Reconcile(ctx context.Context,
 		return reconcile.Result{RequeueAfter: timeout}, nil
 	}
 
+	clusterComputeResourceMoIds, err := common.GetClusterComputeResourceMoIds(ctx)
```

Comment:
Can we avoid calling this function in every Reconcile loop as it queries the AZ CRs by creating a K8s client?

Reply:
yes. placed clusterComputeResourceMoIds in the global section and used it in the Reconcile loop.